synapse/api/errors.py

```diff
@@ -79,6 +79,12 @@ class Codes(str, Enum):
     WEAK_PASSWORD = "M_WEAK_PASSWORD"
     INVALID_SIGNATURE = "M_INVALID_SIGNATURE"
     USER_DEACTIVATED = "M_USER_DEACTIVATED"
+
+    # The account has been suspended on the server.
+    # By opposition to `USER_DEACTIVATED`, this is a reversible measure
+    # that can possibly be appealed and reverted.
+    USER_ACCOUNT_SUSPENDED = "M_USER_ACCOUNT_SUSPENDED"
```

Comment:
In the comment I think we ought to say that this is part of a proposed MSC(?)

Reply:
(Shouldn't this use an unstable prefix?)

Reply:
Do we have unstable prefixes for error codes? Do you have an example at hand?

Reply:
I don't think we have a precedent in using unstable prefixes for error codes (probably because there hasn't been a new one since unstable prefixes have been introduced, but I can be wrong), but it sounds like the kind of things that needs one.

Reply:
So what would this look like?
```python
USER_ACCOUNT_SUSPENDED = "M_ORG_MATRIX_MSC3823_USER_ACCOUNT_SUSPENDED"
```
or something more complicated?

Reply:
> Huh, doesn't seem I used one for [MSC3083](https://github.com/matrix-org/matrix-doc/pull/3083), so maybe it is fine.

I would argue that that's an oversight rather than good practice.

> M_ORG_MATRIX_MSC3823_USER_ACCOUNT_SUSPENDED

Looks good to me (note that it will also need to be mentioned in the MSC).

Reply:
Updated PR and MSC.